src/sqore.ts

```diff
@@ -55,8 +56,10 @@ export class Sqore {
      *
      * @param container HTML element for rendering visualization into.
      * @param renderDepth Initial layer depth at which to render gates.
+     * @param isEditable Optional value enabling/disabling editable feature
+     * @param callbackFn Optional function to trigger when changing elements in circuit
```

Comment:
there can be potentially multiple callbacks, so let's give this one a specific name, like `onChange`, or `onCircuitChange`

Reply:
I agree!

Reply:
009a1d7 change `callbackFn` to `onCircuitChange`